plugins/woocommerce/client/legacy/js/frontend/wp-consent-api-integration.js

```diff
@@ -0,0 +1,24 @@
+( function ( $ ) {
+	'use strict';
+
+	// check init order source attribution on consent change
+	const consentCategory = 'marketing';
+	document.addEventListener( 'wp_listen_for_consent_change', ( e ) => {
+		const changedConsentCategory = e.detail;
+		for ( let key in changedConsentCategory ) {
```

Comment:
I don't see a reason for the key here to not be a `const` ...? 

```suggestion
		for ( const key in changedConsentCategory ) {
```